### PR TITLE
Fixing incoming webhooks

### PR DESCRIPTION
**Pull request by crspeller**
Woopsie.

### CODE REVIEW

File: api/webhook.go [hunk 1]
```diff
@@ -23,6 +26,12 @@ func InitWebhook(r *mux.Router) {
 	sr.Handle("/outgoing/regen_token", ApiUserRequired(regenOutgoingHookToken)).Methods("POST")
 	sr.Handle("/outgoing/delete", ApiUserRequired(deleteOutgoingHook)).Methods("POST")
 	sr.Handle("/outgoing/list", ApiUserRequired(getOutgoingHooks)).Methods("GET")
+
+	sr.Handle("/{id:[A-Za-z0-9]+}", ApiAppHandler(incomingWebhook)).Methods("POST")
```
Comment (hmhealey) [inline api/webhook.go]: What's the difference between the old and new route? I think the manage UI still displays the old one.

**Comment by hmhealey** [comment]
Looks good other than my one question

**Comment by enahum** [comment]
nice, with the submitted PR it now responds to both routes `/hooks/:id` and `/api/v1/hooks/:id` the only thing I would do is change the Frontend to show the new route instead of the old one leaving the old one just to support old versions until everyone has the chance to update.

File: api/webhook.go (hunk 1 above)
Comment (crspeller) [inline api/webhook.go]: @hmhealey No difference between the two routes. (you can see that they have the same handler) We want to switch to the new route, but want to keep supporting the old one for awhile. 
Documentation should be updated to the new route. Ticket: https://mattermost.atlassian.net/browse/PLT-2402

Comment (hmhealey) [inline api/webhook.go]: K makes sense

**Comment by jwilander** [comment]
+1

**Pull request merged**